app/Http/Controllers/Users/LDAPImportController.php

```diff
@@ -6,6 +6,7 @@
 use App\Services\LdapAd;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Artisan;
+use App\Models\User as UserModel; // to more-clearly distinguish between the Users *controller* Namespace, at the top. (it's *not* a conflict, it's Users vs. User)
```

Comment:
Could we not just point it to the model inline?

`$this->authorize('update', \App\Models\User::class);`

I only ask because we don't do this *anywhere* else, and I'm afraid this way might be confusing.

Reply:
We could - and in fact we could instead just refer to `User::class` *after* using that `use App\Models\User;` line, and that *would* work - as that's how the rest of the other controllers in the User namespace do it, and we haven't heard any problems with them.

It's your call. I'm torn between **Consistency** (which is good!) and **Clarity** (which is also good!).

Let me know which one(s) you prefer, and I'll check them out and report back.

Reply:
I think in this case I'd prefer consistency. I'm happy to refactor later to use your method (which is cleaner, IMHO), but *one* exception can cause a lot of confusion in a project this big. :(